src/assets/FA-ICONS.ts

```diff
@@ -0,0 +1,66 @@
+import {
```

Comment:
Maybe we should add a comment here to indicate that every icon from font awesome should be imported here?